mlrun/api/api/endpoints/frontend_spec.py

```diff
@@ -65,6 +65,7 @@ def get_frontend_spec(
         auto_mount_params=config.get_storage_auto_mount_params(),
         default_artifact_path=config.artifact_path,
         default_function_pod_resources=mlrun.mlconf.default_function_pod_resources.to_dict(),
+        default_preemption_mode=mlrun.mlconf.function_defaults.preemption_mode,
```

Comment:
The ticket says `default_function_preemption_mode` and you did `default_preemption_mode`, on purpose ?

Reply:
oopsi